services/site/tailwind.config.js

```diff
@@ -82,6 +82,21 @@ module.exports = {
                 content: "none !important",
               },
             },
+            a: {
+              color: theme("colors.grey.DEFAULT"),
+              textDecorationLine: "none",
+              borderColor: theme("colors.white"),
```

Comment:
The border should have the same color as the text

```suggestion
              borderColor: theme("colors.grey.DEFAULT"),
```